documentation/source/building.rst

```diff
@@ -173,6 +173,12 @@ Environment Variables
 
```

Comment:
On the line above this one , ``` or the presence of :envvar:`NO_COLOR` ```

Reply:
This comment still applies

Reply:
Hi @Botnic, could you add a commit with this change (and one other that I'm going to write now)?

Reply:
@Botnic ping

Reply:
Unresolving, this comment still applies